### PR TITLE
SMS Function Update

### PR DESCRIPTION
**Pull request by jiaqi1993**
Why
Recreated PR.

Created a new Entity SMS with the SMS Function Implemented, Tested with PostMan Working,
But I've removed my password for the sms sending to prevent it from sending without my monitoring.

Postman Test on Agency, IncidentCallReport still working on.
How
implemented SMS using a requests python library.

Side Effects
SMS function working

### CODE REVIEW

File: CZ3003CMS/settings.py [hunk 1]
```diff
@@ -44,7 +44,11 @@
     'IncidentLog',
     'IncidentCallReport',
     'Agency',
+<<<<<<< HEAD
```
Comment (chinjieh) [inline CZ3003CMS/settings.py]: There's a merge conflict (most likely during rebase) that is unresolved here, please resolve!

File: Incident/migrations/0001_initial.py [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # -*- coding: utf-8 -*-
+<<<<<<< HEAD
```
Comment (chinjieh) [inline Incident/migrations/0001_initial.py]: Merge conflict here also

File: SMS/views.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+
+
+		commonHttp.save_model_obj(new_sms)
+
+		url = "http://smsgateway.me/api/v3/messages/send"
+		data = {'email':'xiaojia1993@gmail.com', 'password':'','device':'31268','number': json_obj[expectedAttr["TO"]],'message': json_obj[expectedAttr["TITLE"]] + "," +json_obj[expectedAttr["MESSAGE"]]}
+
+		r=requests.post(url, data)
+		print "test"
```
Comment (chinjieh) [inline SMS/views.py:49]: I think `print` does not actually print to console when heroku server is running. Instead, it gets 'eaten' up and displayed only after server is shutdown, which is bad.

I actually setup a `logger` configuration, you can reference `IncidentLocation` to see how debug messages are logged. I think you use `logger.debug(_)` to log to console.

But anyway, if everything is working, we should not be posting debug logs to console anyway.

**Review by chinjieh — CHANGES_REQUESTED** [review]
Stuff to change!

File: SMS/views.py (hunk 3 above)
Comment (chinjieh) [reply SMS/views.py:49]: Since we're on this PR, can settle this also?

**Comment by chinjieh** [comment]
Oh oops I tried to fix the merge conflict with `master` and this branch, apparently Github automatically detects the PR as merged and closed. Ohwell, learning point for the future haha.

File: SMS/views.py (hunk 3 above)
Comment (chinjieh) [reply SMS/views.py:49]: Ok nevermind, since I accidentally merged it in. Could @jiaqi1993 settle this locally?

Comment (jiaqi1993) [reply SMS/views.py:49]: Ok

**Pull request merged**